### PR TITLE
adding Spring Security logo

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
springsecurity

**Issue:**
Adding Spring security logo: https://spring.io/projects/spring-security

### Description
Adding this one from existing svg on https://spring.io/images/projects/spring-security-b712a4cdb778e72eb28b8c55ec39dbd1.svg, which is the projects page for Spring. 
Spring trademark guidelines apply: same colors, same uses.
Added as part of the Spring ecosystem alongside Spring Framework and Spring boot, which already exist. It also has 5k+ stars on github: https://github.com/spring-projects/spring-security

### CODE REVIEW

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @jorgeamadosoria for your contribution 💯 🔑 
I made minor comments. Can you also revert the changes in `package-lock.json`?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8592,6 +8592,11 @@
             "hex": "6DB33F",
             "source": "https://spring.io/projects"
         },
+        {
+            "title": "Spring Security",
+            "hex": "6DB33F",
```
[the remark this answers is not in the document]
Comment (jorgeamadosoria) [reply _data/simple-icons.json:8597]: From the other spring projects. I believe they all should have the same green.

[the remark this answers is not in the document]
Comment (service-paradis) [reply _data/simple-icons.json:8597]: Well... looks like the other icons on this page uses `#6DB33F`, while Spring security use `#6BB344`. Lets keep it this way.

[the remark this answers is not in the document]
Comment (jorgeamadosoria) [reply _data/simple-icons.json:8597]: going by the color here: https://spring.io/trademarks
the trademark color is indedd `#6DB33F`. I guess the fill color in Spring Security is a mistake, or there is no established green shade for Spring projects.
We can do whatever, but for the sake of consistency I would stick with `#6DB33F` even if it contradicts the color in the source.

[the remark this answers is not in the document]
Comment (service-paradis) [reply _data/simple-icons.json:8597]: Yep, this what I'm saying!

**Review by service-paradis — APPROVED** [review]
LGTM 💯 Thank you @jorgeamadosoria!

**Pull request merged**